### ISSUE TITLE
docker installations have issues with mss-post-link.sh

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#207]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/207) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

the current pipeline will install mss for getting all packages involved .

Something in mss-post-link.sh is incompatible

```
#!python

ERROR conda.core.link:_execute_actions(335): An error occurred while installing package 'conda-forge::mss-1.5.1-py27_0'.
LinkError: post-link script failed for package conda-forge::mss-1.5.1-py27_0
running your command again with `-v` will provide additional information
location of failed script: /opt/conda/bin/.mss-post-link.sh
```

==> script messages <==

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

This is related to #211 and fixed at the moment the mss docker image is on the docker hub.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

this had to be reproduced with a python 3 docker setup, therefore it is moved to a minor

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

there is not an issue if docker image is created in a directory where the meta.yaml and the shellscripts are

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

on a docker with no prebuild image this failure occures

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

scripts misses execute bit

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Docker setup updated to python 3, fixes #207, #333

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss/stable (pull request #550)

Docker setup updated to python 3, fixes #207, #333
